observability/otel/otel-collector/otel-collector-exporters.adoc

```diff
@@ -290,6 +291,42 @@ include::snippets/technology-preview.adoc[]
 <6> Disables verifying the server's certificate chain and host name. The default is `+false+`.
 <7> ServerName indicates the name of the server requested by the client to support virtual hosting.
 
+[id="file-exporter_{context}"]
+== File Exporter
+
+The File Exporter writes telemetry data to files in persistent storage and supports file operations such as rotation, compression, and writing to multiple files. With this exporter, you can also use a resource attribute to control file naming. The only required setting is `path`, which specifies the destination path for telemetry files in the persistent-volume file system.
+
+:FeatureName: The File Exporter
+include::snippets/technology-preview.adoc[]
+
+.OpenTelemetry Collector custom resource with the enabled File Exporter
+[source,yaml]
+----
+# ...
+  config: |
+    exporters:
+      file:
+        path: /data/metrics.json # <1>
+        rotation: # <2>
+          max_megabytes: 10 # <3>
+          max_days: 3 # <4>
+          max_backups: 3 # <5>
+          localtime: true # <6>
+        format: proto # <7>
+        compression: zstd # <8>
+        flush_interval: 5 # <9>
+# ...
+----
```

Comment:
Does this [include the full hierarchy](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#yaml-formatting-for-kubernetes-and-openshift-api-objects) to these values? I see that it matches the others. If it is not the full hierarchy, consider updating them all later to show it. If it is the full hierarchy, ignore me :nerd_face: